test/transaction.cpp

```diff
@@ -0,0 +1,230 @@
+			BOOST_CHECK_MESSAGE(txFromFields.gasPrice() == txFromRlp.gasPrice(), "GasPrice in given RLP not matching the Transaction gasPrice!");
+			BOOST_CHECK_MESSAGE(txFromFields.gas() == txFromRlp.gas(),"Gas in given RLP not matching the Transaction gas!");
+			BOOST_CHECK_MESSAGE(txFromFields.nonce() == txFromRlp.nonce(),"Nonce in given RLP not matching the Transaction nonce!");
+			BOOST_CHECK_MESSAGE(txFromFields.receiveAddress() == txFromRlp.receiveAddress(), "Receive address in given RLP not matching the Transaction 'to' address!");
+			BOOST_CHECK_MESSAGE(txFromFields.sender() == txFromRlp.sender(), "Transaction sender address in given RLP not matching the Transaction 'vrs' signature!");
+			BOOST_CHECK_MESSAGE(txFromFields == txFromRlp, "However, txFromFields != txFromRlp!");
+			BOOST_REQUIRE (o.count("sender") > 0);
+
+			Address addressReaded = Address(o["sender"].get_str());
+			BOOST_CHECK_MESSAGE(txFromFields.sender() ==  addressReaded || txFromRlp.sender() == addressReaded, "Signature address of sender does not match given sender address!");
+		}
+
+		if(_fillin == true)
+		{
+			BOOST_REQUIRE(o.count("transaction") > 0);
+			mObject tObj = o["transaction"].get_obj();
+
+			//Construct Rlp of the given transaction
+			RLPStream rlpStream;
+			rlpStream.appendList(tObj.size());
+
+			if(tObj.count("nonce") > 0)
+				rlpStream << bigint(tObj["nonce"].get_str());
+
+			if(tObj.count("gasPrice") > 0)
+				rlpStream << bigint(tObj["gasPrice"].get_str());
+
+			if(tObj.count("gasLimit") > 0)
+				rlpStream << bigint(tObj["gasLimit"].get_str());
+
+			if(tObj.count("to") > 0)
+			{
+				if (tObj["to"].get_str().empty())
+					rlpStream << "";
+				else
+					rlpStream << Address(tObj["to"].get_str());
+			}
+
+			if(tObj.count("value") > 0)
+				rlpStream << bigint(tObj["value"].get_str());
+
+
+			if(tObj.count("data") > 0)
+				rlpStream << importData(tObj);
+
+			if(tObj.count("v") > 0)
+				rlpStream << bigint(tObj["v"].get_str());
+
+			if(tObj.count("r") > 0)
+			{
+				u256 r = h256(fromHex(tObj["r"].get_str()));
+				rlpStream << r;
+			}
+
+			if(tObj.count("s") > 0)
+			{
+				u256 s = h256(fromHex(tObj["s"].get_str()));
+				rlpStream << s;
+			}
+
+			if(tObj.count("extrafield") > 0)
+				rlpStream << bigint(tObj["extrafield"].get_str());
+
+			o["rlp"] = "0x" + toHex(rlpStream.out());
+
+			try
+			{
+				Transaction txFromFields(rlpStream.out(), true);
+				if (!txFromFields.signature().isValid())
+					BOOST_THROW_EXCEPTION(Exception() << errinfo_comment("transaction from RLP signature is invalid") );
+
+				o["sender"] = toString(txFromFields.sender());
+			}
+			catch(...)
+			{
+				o.erase(o.find("transaction"));
+			}
+		}
+	}//for
+}//doTransactionTests
+
+} }// Namespace Close
+
+
+BOOST_AUTO_TEST_SUITE(TransactionTests)
+
+BOOST_AUTO_TEST_CASE(ttFillerTest)
+{
+	dev::test::executeTests("ttTransactionTest", "/TransactionTests", dev::test::doTransactionTests);
```

Comment:
I use the same name for test (ttFillerTest) and File (ttTransactionTest in your case). tt already specifies that this ias transaction test.